wherobots/db/constants.py

```diff
@@ -56,5 +56,25 @@ class ResultsFormat(LowercaseStrEnum):
     ARROW = auto()
 
 
+DEFAULT_RESULTS_FORMAT: ResultsFormat = ResultsFormat.ARROW
+"Use efficient Arrow format for results by default."
+
+
 class DataCompression(LowercaseStrEnum):
     BROTLI = auto()
+
+
+DEFAULT_DATA_COMPRESSION: DataCompression = DataCompression.BROTLI
```

Comment:
I'd prefer to leave the defaults up to server + protocol_version to figure out. Easier to sunset formats/compression when needed, or switch to more optimal ones as long as protocol supports it. WDYT?